### DISCUSSION TITLE
Noob Question: Add more Audio tracks to Host Audio (Ableton Live)

### DISCUSSION THREAD

**Discussion by TheTwiddler** [DISTRHO/Cardinal#308] [Q&A]
Hi its my first post in GitHub and at first i wantet to say thank you for the great work! I was waiting for that to happen and i think a lot of other musicians and synthesizerenthusiasts are just happy to use VCV in a DAW.

I tried to find the Information in the manual maybe i missed it but how can i add several stereo Audio tracks to a patch for crossmodulation etc.?

**Comment by falkTX** [comment]
You can't in Ableton, at least for now.
If you check the readme, it clearly says FX and Synth variants are stereo.

The main variant has more IO, but it does not work in Ableton yet.
VST3 bus layouts are super weird, I have not done that side of the plugin implementation yet.

**Reply by TheTwiddler** [reply]
Thanks for the fast reply.
